### DISCUSSION TITLE
Use of ckanext.validation

### DISCUSSION THREAD

**Discussion by wilsonmilagres** [ckan/ckan#6412] [General]
Hello,

I need to use the ckanext.validation plug-in (https://github.com/frictionlessdata/ckanext-validation), but I'm having a lot of problems with it, including doing a validation_optinons and it doesn't respect my settings.
Has anyone worked with him and would he know how to give me some tips?

**Comment by amercader** [comment]
@wilsonmilagres I haven't worked on ckanext-validation in a while but just this week I had to assess the work involved in making it work on CKAN 2.9 (quite a lot)

But if you are on CKAN 2.8 maybe I can help if you give us more details about the actual issues you are having
